src/engine/engine.js

```diff
@@ -75,6 +75,8 @@ class Engine {
     Engine.instance = this;
     this.emitter = emitter;
     if (heartBeat) {
+      Engine.instance.beat_instances = process.env.HEARTBEAT_INSTANCES || 'TIMER,PROCESS';
```

Comment:
Não gosto de guardar listas como variáveis de ambiente, ainda mais sendo um enum, chance de erro bem grande (espaços, maisculas e minusculas, etc).
Eu iria por 2 alternativas:
1. Usar o batch, afinal se ele definir >0 poderíamos pressupor a vontade de ter a instância executando. Nesse caso removeríamos o valor default, se a propriedade for 0 ou undefined, significaria não executar a instância.
2. Criar um boleano TIMER_INSTANCE, PROCESS_INSTANCE. Nesse caso a execução da instância dependeria de 2 parâmetros, o boleano E o batch. Para que não fiquemos com 2 propriedades obrigatórias, teríamos o valor default caso do usuário não definisse o tamanho do batch.